frontend/src/main/java/puf/frisbee/frontend/viewmodel/GameViewModel.java

```diff
@@ -91,7 +101,7 @@ private void startCharacterAnimation() {
 			@Override
 			public void handle(long l) {
 				// moving is not possible once the level is over
-				if (showLevelSuccessDialog.getValue()) return;
+				if ((getTeamLives() == 0) || showLevelSuccessDialog.getValue()) return;
```

Comment:
Wenn man `getTeamLives() == 0)` auslagert in eine Function `teamHasLivesLeft`(die true oder false zurückgibt), dann kann das auch unten bei den ifs benutzt werden.

Reply:
OK, guter Punkt.